bridge/bridgeapp/views.py

```diff
@@ -47,26 +52,38 @@ def get(self, request, category_id, category_slug):  #category_slug needed for U
     def post(self, request, category_id, category_slug):
```

Comment:
Please read my review (overall) comments. I can't go further at this rate.

Long and numerous comments can serve to obscure the code from which readers are trying to make sense in the shortest time possible.